honeybee_radiance/view.py

```diff
@@ -31,30 +33,24 @@ class View(object):
             print(g)
 
         > -vtv -vp 0.000 0.000 0.000 -vd 0.000 0.000 1.000 -vu 0.000 1.000
-           0.000 -vh 29.341 -vv 32.204 -x 300 -y 300 -vs -0.500 -vl -0.500
-           -vo 100.000
+           0.000 -vh 29.341 -vv 32.204  -vs -0.500 -vl -0.500 -vo 100.000
 
         > -vtv -vp 0.000 0.000 0.000 -vd 0.000 0.000 1.000 -vu 0.000 1.000
-           0.000 -vh 29.341 -vv 32.204 -x 300 -y 300 -vs 0.500 -vl -0.500
-           -vo 100.000
+           0.000 -vh 29.341 -vv 32.204 -vs 0.500 -vl -0.500 -vo 100.000
 
         > -vtv -vp 0.000 0.000 0.000 -vd 0.000 0.000 1.000 -vu 0.000 1.000
-           0.000 -vh 29.341 -vv 32.204 -x 300 -y 300 -vs -0.500 -vl 0.500
-           -vo 100.000
+           0.000 -vh 29.341 -vv 32.204 -vs -0.500 -vl 0.500 -vo 100.000
 
         > -vtv -vp 0.000 0.000 0.000 -vd 0.000 0.000 1.000 -vu 0.000 1.000
-          0.000 -vh 29.341 -vv 32.204 -x 300 -y 300 -vs 0.500 -vl 0.500
-          -vo 100.000
+          0.000 -vh 29.341 -vv 32.204 -vs 0.500 -vl 0.500 -vo 100.000
     """
 
-    VIEWTYPES = {0: 'vtv', 1: 'vth', 2: 'vtl', 3: 'vtc', 4: 'vta', 5: 'vts'}
-
-    def __init__(self, name, origin=None, direction=None, up_vector=None, type=0,
-                 h_size=60, v_size=60, shift=0, lift=0):
+    def __init__(self, name, position=None, direction=None, up_vector=None, type='v',
+                 h_size=60, v_size=60, shift=None, lift=None):
         u"""Create a view.
 
         Arg:
-            origin: Set the view origin location (-vp) to (x, y, z). This is the focal
+            position: Set the view position location (-vp) to (x, y, z). This is the focal
```

Comment:
`position location`  sounds a bit redundant.  I think you can just say `position`